mac

```diff
@@ -159,27 +158,48 @@ fancy_echo "Update heroku binary..."
 brew unlink heroku
 brew link --force heroku
 
-fancy_echo "Configuring Ruby ..."
-find_latest_ruby() {
-  rbenv install -l | grep -v - | tail -1 | sed -e 's/^ *//'
+fancy_echo "Configuring asdf version manager..."
+if [ !  -d "$HOME/.asdf" ]; then
+  git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.3.0
+  append_to_zshrc "source $HOME/.asdf/asdf.sh" 1
```

Comment:
What do you think about not doing the `append_to_zshrc`? Relying instead on the `zsh/configs/post/path.zsh` approach in https://github.com/thoughtbot/dotfiles/pull/534?

I've been feeling like the existing `append_to_zshrc 'eval "$(rbenv init - --no-rehash)"' 1` should probably be removed or replaced by something that operates on `zsh/configs/post/path.zsh` instead of `zshrc` so all the enabling of shims and autocompletion is co-located and easier to reason about / debug.